docs/resources/dashboard_share.md

```diff
@@ -0,0 +1,90 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "squaredup_dashboard_share Resource - terraform-provider-squaredup"
+subcategory: ""
+description: |-
+  Enable Open Access for a dashboard
+---
+
+# squaredup_dashboard_share (Resource)
+
+Enable Open Access for a dashboard
+
+## Example Usage
+
+```terraform
+resource "squaredup_workspace" "application_workspace" {
+  display_name        = "Application Team"
+  description         = "Workspace with Dashboards for Application Team"
+  open_access_enabled = true
+}
+
+resource "squaredup_dashboard" "sample_dashboard" {
+  dashboard_template = <<EOT
+{
+  "_type": "layout/grid",
+  "contents": [
+    {
+      "x": 0,
+      "h": 2,
+      "i": "1",
+      "y": 0,
+      "config": {
+        "title": "",
+        "description": "",
+        "_type": "tile/text",
+        "visualisation": {
+          "config": {
+            "content": "Sample Tile",
+            "autoSize": true,
+            "fontSize": 16,
+            "align": "center"
+          }
+        }
+      },
+      "w": 4
+    }
+  ],
+  "columns": 1,
+  "version": 1
+}
+EOT
+  workspace_id       = squaredup_workspace.application_workspace.id
+  display_name       = "Sample Dashboard"
+}
+
+resource "squaredup_dashboard_share" "sample_dashboard_share" {
+  dashboard_id           = squaredup_dashboard.sample_dashboard.id
+  workspace_id           = squaredup_workspace.application_workspace.id
+  require_authentication = true
+  enable_link            = true
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `dashboard_id` (String) The ID of the dashboard
+- `require_authentication` (Boolean) If false, the dashboard will be accessible to anyone with the link
+- `workspace_id` (String) The ID of the workspace where the dashboard is located
+
+### Optional
+
+- `enable_link` (Boolean) If false, the dashboard will not be accessible via Open Access Link
```

Comment:
I'd simplify to "enabled" and use "(Boolean) If false, sharing of the dashboard is disabled"